TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h

```diff
@@ -18,6 +18,7 @@
 @property (nonatomic, weak) NSURL *_Nullable linkTeamsResults;
 @property (nonatomic, weak) NSURL *_Nullable linkMe;
 @property (nonatomic, weak) NSURL *_Nullable linkMemberRegistrationSignups;
+@property (nonatomic, weak, nullable) NSURL * linkHoyle;
```

Comment:
For quick readability, do we want to keep the definitions, at least in this file, standard? either conforming this to the other definitions `property (nonatomic, weak) NSURL *_Nullable`, or search/replace (in this file) so they all match `@property (nonatomic, weak, nullable) NSURL *`. 

As it is it looks like `linkHoyle` is special.

Reply:
I almost changed them all, but decided against it right before committing.

Reply:
Went ahead and added full nullability to this file